### ISSUE TITLE
Would it be possible to get `trackio` experiment training logs for the finetuning runs?

### ISSUE THREAD

**Issue by abidlabs** [thinking-machines-lab/tinker-cookbook#6]
Thanks for the release of Tinker! Would you be open to adding a [Trackio](https://github.com/gradio-app/trackio) integration for logging and monitoring fine-tuning runs with Tinker?

Having access to training metrics would help researchers compare different runs (e.g. easily compare multiple fine-tuning experiments with different hyperparameters), debug training issues (identify problems like overfitting or gradient explosions early), and generally provide valuable insights to researchers to optimize their fine-tuning workflows.

**Comment by Tiiiger** [comment]
Hi @abidlabs

Great suggestion!

We currently have a wandb integration and can be turned on in every recipe example. Will need to think a bit more between balancing cook book complexity and more integrations.

**Comment by abidlabs** [comment]
Thanks for considering @Tiiiger! One benefit of Trackio is that it can run completely locally (without needing to make an account, etc.)

**Comment by Tiiiger** [comment]
hi @abidlabs if you want to follow the pattern in #13 and help contribute a PR, I can help review and merge it

**Comment by abidlabs** [comment]
thanks @Tiiiger here ya go! https://github.com/thinking-machines-lab/tinker-cookbook/pull/15
